### ISSUE TITLE
Videolab Input Node

### ISSUE THREAD

**Issue by cemillerena** [teenageengineering/videolab#16]
Can't seem to get the VIdeolab Input Note to work as stated on the Wiki. Tempo is not been transmitted. OP-Z Midi settings are alright, but still can't speed modulation to work.

**Comment by finnTE** [comment]
VideolabInput only works inside the app, OP-Z does not currently issue the videolab sysex events. Will update wiki.

**Comment by sourya-sen** [comment]
Having trouble with this too - makes life difficult that I can’t troubleshoot within Unity and need the app. Active Pattern, Track and Project - are these outputting ints or floats? I have something like this set up - which works by changing the float output within Unity, but changing the Active Pattern while running it within the app doesn’t do anything…

Could it have something to do with limitations within the App? Do custom scripts not work, either?

<img width="1454" alt="Screenshot 2019-07-05 at 4 05 10 PM" src="https://user-images.githubusercontent.com/22247166/60717187-cc8b9800-9f3e-11e9-81be-f579f7892099.png">

**Comment by momo-the-monster** [comment]
That's right @sourya-sen, no custom scripts are allowed since the AssetBundle contains only new content.
